src/ir_Pioneer.cpp

```diff
@@ -0,0 +1,101 @@
+// Copyright 2009 Ken Shirriff
+// Copyright 2017 David Conran
+// Copyright 2018 Kamil Palczewski
+
+#define __STDC_LIMIT_MACROS
+#include <stdint.h>
+#include <algorithm>
+#include "IRrecv.h"
+#include "IRsend.h"
+#include "IRutils.h"
+#include <Arduino.h>
+
+//                        PPPP  III  OOO  N   N EEEE EEEE RRRR
+//                        P   P  I  O   O NN  N E    E    R   R
+//                        PPPP   I  O   O N N N EEE  EEE  RRRR
+//                        P      I  O   O N  NN E    E    R R
+//                        P     III  OOO  N   N EEEE EEEE R  RR
+
+// NEC originally added from https://github.com/shirriff/Arduino-IRremote/
+
+// Constants
+// Ref:
+//  http://www.sbprojects.com/knowledge/ir/nec.php
+
+const uint16_t kNecTick = 560;
```

Comment:
there is no `ir_NEC.h`, so I'm a little confused. Should like rewrite `ir_NEC.cpp`, so it would include variables from `ir_NEC.h` and `ir_Pioneer.cpp` should use those variables as well?

Reply:
Yeah. I'll probably need to do that. But for now, assume my comment about using `ir_NEC.h` for now is wrong/bad. I clearly had a brain fade. I'll split it out and clean this up later.